projects/plugins/jetpack/modules/search/instant-search/components/search-result-expanded.scss

```diff
@@ -87,3 +83,48 @@ $image-square-size: 128px;
 .jetpack-instant-search__search-result-expanded__image {
 	border-radius: 5px;
 }
+
+.jetpack-instant-search__search-result-expanded__content-container {
+	display: flex;
+}
+
+.jetpack-instant-search__search-result-expanded__footer {
+	list-style-type: none;
+	margin-top: 10px;
+	margin-left: 0;
```

Comment:
Do we need to support non-P2 themes? If not, feel free to close out this comment.

If we do, however, we'll want to add `padding-left: 0;` to properly support the Twenty Twenty-one theme.

Reply:
We don't have to support other themes for now but I will add the padding.